util/chplenv/chpl_mem.py

```diff
@@ -37,6 +37,10 @@ def get(flag='host'):
             mem_val = 'jemalloc'
     else:
         error("Invalid flag: '{0}'".format(flag), ValueError)
+
+    var_name = 'CHPL_{0}_MEM'.format(flag.upper())
+    var_name = 'CHPL_MEM' if chpl_target_mem is None else var_name
```

Comment:
Same situation as with my "something seems off here" comment for chpl_jemalloc.py. IMO this line should just be removed b/c CHPL_MEM is deprecated (judging from above code)